### DISCUSSION TITLE
BT815 adding tags to kys widget

### DISCUSSION THREAD

**Discussion by fredykoci** [RudolphRiedel/FT800-FT813#86] [Q&A]
Hi, do you know how to add screen touch tag value to keys widget?

**Comment by RudolphRrr** [comment]
Hi,

the CMD_KEYS widget uses the ASCII values of the keys as the touch tags.

You can see that in EVE Screen Editor, when you pull in the "Keys" widget you get a list of commands added to the display list.
This is on the bottom left in the "Inspector" tab.
